pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -331,16 +331,25 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 		}
 	}
 
+	optionaUILayerNamespaces := []string{render.GuardianNamespace}
```

Comment:
I think this needs a comment explaining why these are optional and what a UI layer is at all.

Reply:
Added a comment and updated the variable name as sgLayerTigeraNameSpaces